.github/BOTMETA.yml

```diff
@@ -151,8 +151,16 @@ files:
   $filters/jc.py:
     maintainers: kellyjonbrazil
   $filters/json_query.py: {}
+  $filters/lists.py:
+    maintainers: cfiehe
+  $filters/lists_difference.yml:
+    maintainers: cfiehe
+  $filters/lists_intersect.yml:
+    maintainers: cfiehe
   $filters/lists_mergeby.py:
     maintainers: vbotka
+  $filters/lists_union.yml:
+    maintainers: cfiehe
```

Comment:
@felixfontein Should I add myself here as maintainer?

Reply:
Yes :)